### ISSUE TITLE
Add more examples? (actix-web, std-async)

### ISSUE THREAD

**Issue by IronOxidizer** [kyren/webrtc-unreliable#14]
Are you accepting PRs for more examples? I think having one using `actix-web` + `std-async` would be beneficial.

I created such an example here: https://github.com/IronOxidizer/serverless-webrtc-rust

**Comment by kyren** [comment]
Yeah I should be accepting PRs for examples but I've just been very very busy lately.  I'm sorry it took me so long to respond.

I'm at least going to steal your idea of getting rid of the STUN and TURN servers, that's much better.

**Comment by IronOxidizer** [comment]
No worries. I'll see if I can get a PR in by the end of the week and close up this issue.

Awesome project btw, very useful.
